conda/meta.yaml

```diff
@@ -14,14 +14,14 @@ build:
 
 requirements:
     build:
-        - python 2.7|>=3.4
+        - python
         - setuptools
         - nomkl # [not win]
         - future >=0.14
         - numpy >=1.9
         - scipy >=0.14
     run:
-        - python 2.7|>=3.4
+        - python
```

Comment:
The problem is that it pulls in 3.6 during build even if I specify `--python 3.5`. Not sure if it's a conda bug but with this setup the correct version is pulled in.